src/components/AppToolbar.tsx

```diff
@@ -17,6 +17,8 @@ import maputnikLogo from 'maputnik-design/logos/logo-color.svg?inline'
 import { withTranslation, WithTranslation } from 'react-i18next';
 import { supportedLanguages } from '../i18n';
 
+import Dropzone from 'react-dropzone'; // for class components
```

Comment:
```suggestion
import Dropzone from 'react-dropzone';
```
Is this comment meaningful?

Reply:
This came from `react-dropzone` docs. Functional/class components require different imports (useDropzone/Dropzone respectively)

Reply:
Yes, please.